shared/js/initApp.js

```diff
@@ -172,21 +172,64 @@ dashboard.saveProject = function(callback) {
       location.hash = dashboard.currentApp.id = data.id;
       callbackSafe(callback, data);
     });
-    dashboard.showProjectHeader();
+    if (!dashboard.currentApp.hidden) {
```

Comment:
(to clarify, you're saying this entire call to showProjectHeader is unnecessary? I think you're right, I'll try it out)